diff --git a/dist/esm/src/webhook/index.js b/dist/esm/src/webhook/index.js
index 88a91b235aeb6b3e592623bb1d583320afde92cf..ccc043c839ba717847e3bece6bcf1d5c20346193 100644
--- a/dist/esm/src/webhook/index.js
+++ b/dist/esm/src/webhook/index.js
@@ -1,5 +1,5 @@
 import { WhatsappApiNotificationPayloadSchemaType } from "./schema";
-import * as express from "express";
+import express from "express";
 import { json as expressJson, } from "express";
 import { InteractionNotificationTypeEnum, MessageStatusEnum, NotificationMessageTypeEnum, SystemNotificationTypeEnum, } from "./type";
 import { TextMessageEvent } from "./events/text";
